### PR TITLE
OPS: Update RN

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zwglkf3jiv4kvswhactgvyxmri?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10885)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/kmdh24gg7fmn5tezpmok44qq3i?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10887)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/com6ejc7r63isiu4porzcmhbnu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10892)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/35wvnyw5pvs2pekowwefjz4h3m?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10893)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/y65l2sni2xwajis45ynzy5gwly?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10898)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/53lx2mw72l6jfp7k2xzrmi5c7u?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10903)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/vffgzireddmx6hxt7fjirzafvm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10905)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ubomqzmem5ub734my25nigzabi?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10906)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/jha7tigfeozyokvm7m6iq56zbu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10908)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/eyngdhnr7byipurlgmtxdzifvq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10910)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zm752xp2qfdizd3gu453iyvj4q?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10912)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/m2me46haurra2dkjdm5wuxilfy?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10915)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zhev4rs4dgzfcl6btdaltettdq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10916)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zwy6koann7swya6s4fyv34plni?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10918)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: package.json [hunk 1]
```diff
@@ -172,21 +175,21 @@
...
     "react-native-safe-area-context": "3.4.1",
-    "react-native-screens": "3.19.0",
-    "react-native-secure-key-store": "https://github.com/BlueWallet/react-native-secure-key-store#63ab38c9d382a819844a086a69cc204c46aa93f9",
-    "react-native-share": "7.9.1",
-    "react-native-svg": "12.4.4",
+    "react-native-screens": "^3.20.0",
+    "react-native-secure-key-store": "https://github.com/BlueWallet/react-native-secure-key-store#2076b48",
+    "react-native-share": "^8.2.1",
+    "react-native-svg": "^13.8.0",
     "react-native-tcp-socket": "5.6.2",
```
[the remark this answers is not in the document]
Comment (marcosrdz) [reply package.json]: Addressed.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xvpsse3lmybepxlyswu25xp6ya?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10919)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Pull request merged**